compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -397,8 +397,6 @@ E0668: include_str!("./error_codes/E0668.md"),
 E0669: include_str!("./error_codes/E0669.md"),
 E0670: include_str!("./error_codes/E0670.md"),
 E0671: include_str!("./error_codes/E0671.md"),
-E0687: include_str!("./error_codes/E0687.md"),
```

Comment:
Please don't remove them from the listing.

Reply:
(I'll add a tidy check for that one too :smile: )

Reply:
oops!

Reply:
Just so you know: https://github.com/rust-lang/rust/pull/93898 :)